### DISCUSSION TITLE
12 word mnemonic phrase support

### DISCUSSION THREAD

**Discussion by jekeam** [Chia-Network/chia-blockchain#12225] [Support]
Hello, I'm a Nucle wallet user and would like to import my wallets into your desktop wallet, but I can't do that because it allows you to import a mnemonic phrase - only 24 words, when will 12 words be supported?

**Comment by shaneo257** [comment]
it wont
what you'll need to do is create a new chia wallet (if you haven't ready done so), save the 24 words somewhere save, and then deposit the xch from the nucle wallet to the wallet address created in the chia client

**Reply by jekeam** [reply]
It's genius, but not practical...
Believe me, I need it, as do many other users, read the tweets where they talk about the NFT.
